core/blockchain_reader.go

```diff
@@ -343,3 +348,31 @@ func (bc *BlockChain) SubscribeAcceptedLogsEvent(ch chan<- []*types.Log) event.S
 func (bc *BlockChain) SubscribeAcceptedTransactionEvent(ch chan<- NewTxsEvent) event.Subscription {
 	return bc.scope.Track(bc.txAcceptedFeed.Subscribe(ch))
 }
+
+func (bc *BlockChain) GetFeeConfigAt(parent *types.Header) (commontype.FeeConfig, error) {
```

Comment:
can we add a comment and make sure the desired behavior is well defined in the comment? Ie. do we assume that the fee config must be defined in the state if it's been enabled or do we fall back to the original config if it's not present in the state?